### PR TITLE
Add ReactiveGit WPF sample app

### PR DESCRIPTION
**Pull request by worldbeater**
@glennawatson made [an app covering almost all ReactiveUI features](https://github.com/glennawatson/ReactiveGit). I've learned a lot while reading it's source code. Wondering why isn't it on the samples page yet :) Thank you Glenn for such a great open-source app. 

Also I deliberately removed the "FirstStepsRUI" app because it's outdated and no longer maintained.

### CODE REVIEW

**Comment by glennawatson** [comment]
Hmm would prefer to upgrade to rxui 9 before we merge this. Also worth mentioning it's incomplete application.

**Comment by glennawatson** [comment]
Updated to use ReactiveUI 9.4 now.

File: input/docs/resources/samples.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Order: 15
 
 * **Dynamic.Trader** Additionally you absolutely must take https://github.com/RolandPheasant/Dynamic.Trader for a test-drive. So so good. 
 
-* **FirstStepsRUI**  ReactiveUI Binding and Routing tutorial covering all the `ReactiveUI.WPF` basics https://github.com/kondaskondas/FirstsStepsRUI
+* **ReactiveGit** — ReactiveUI open-source app covering all ReactiveUI features including Binding and Routing https://github.com/kondaskondas/FirstsStepsRUI
```
Comment (glennawatson) [inline input/docs/resources/samples.md]: Doesn't reference my framework :)

Comment (worldbeater) [reply input/docs/resources/samples.md]: Whoops sorry!

**Pull request merged**